### DISCUSSION TITLE
Testing code lenses

### DISCUSSION THREAD

**Discussion by firelizzard18** [golang/vscode-go#1730] [Discussions]
Now that the testing API is merged, should we remove the `run test` code lens? The `run test` code lens and the play button serve the same purpose. The `debug test` code lens should remain until the test controller can debug test. Of course, instead of actually removing the code lens, we can add a `showRunTestLens` setting that defaults to false. /cc @hyangah 

![image](https://user-images.githubusercontent.com/879055/131228295-2db985a3-93be-47b1-b7df-d93f1305e742.png)

**Comment by codeliger** [comment]
I feel stupid i never would have noticed the play button without seeing this discussion.
Officially disabled code lens today!
